serverless/README.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: pattern
+title: serverless
+folder: serverless
+permalink: /patterns/serverless/
+categories: Architectural
+tags:
+ - Java
+ - Difficulty-Intermittent
+---
+
+## Serverless
+
+Serverless eliminates the need to plan for infrastructure and let's you focus on your 
+application. 
+
+Following are optimization katas you should be aware of while building a serverless 
+applications
+
+* The Lean function
+    * Concise logic
+    * Efficient/single purpose code
+    * ephemeral environment
+* Eventful Invocations
+    * Succinct payloads
+    * resilient routing
+    * concurrent execution
+* Coordinated calls
+    * Decoupled via APIs
+    * scale-matched downstream
+    * secured
+* Serviceful operations
+    * Automated operations
+    * monitored applications
+    * Innovation mindset
```

Comment:
Any links that would explain more about these katas? The terms alone are not enough.